### PR TITLE
TELCODOCS-1358: Clarification in Enabling the assisted service ZTP section

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.12, 4.13., 4.14

Issue: https://issues.redhat.com/browse/TELCODOCS-1358

Link to docs preview: https://64883--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster#enabling-assisted-installer-service-on-bare-metal_ztp-preparing-the-hub-cluster

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jgato** [comment]
lgtm

**Comment by amolnar-rh** [comment]
@jgato contributed to this via [PR](https://github.com/amolnar-rh/openshift-docs/pull/37)

File: modules/ztp-enabling-assisted-installer-service-on-bare-metal.adoc [hunk 1]
```diff
@@ -3,20 +3,22 @@
 // * scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc
 
 [id="enabling-assisted-installer-service-on-bare-metal_{context}"]
-= Enabling the assisted service and updating AgentServiceConfig on the hub cluster
+= Enabling the assisted service
 
-{rh-rhacm-first} uses the assisted service to deploy {product-title} clusters. The assisted service is deployed automatically when you enable the MultiClusterHub Operator with Central Infrastructure Management (CIM). When you have enabled CIM on the hub cluster, you then need to update the `AgentServiceConfig` custom resource (CR) with references to the ISO and RootFS images that are hosted on the mirror registry HTTP server.
+{rh-rhacm-first} uses the assisted service to deploy {product-title} clusters. The assisted service is deployed automatically when you enable the MultiClusterHub Operator on {rh-rhacm-first}. After that, you need to configure the `Provisioning` resource to watch every `Namespace` and to update the `AgentServiceConfig` custom resource (CR) with references to the ISO and RootFS images that are hosted on the mirror registry HTTP server.
```
Comment (JoeAldinger) [inline modules/ztp-enabling-assisted-installer-service-on-bare-metal.adoc:8]: I would maybe try to keep the use of literals the same in the PR.  On line 20, you have `Provisioning` resource and `...to watch all namespaces`. I'd make like 8 and line 20 consistent across their use of back-ticks. For me, line 20's language of `watch all namespaces` reads cleaner than line 8's `every Namespace`.

File: scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc [hunk 2]
```diff
@@ -39,6 +39,12 @@ include::modules/ztp-acm-adding-images-to-mirror-registry.adoc[leveloffset=+1]
 
 include::modules/ztp-enabling-assisted-installer-service-on-bare-metal.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#enable-cim[Enabling the Central Infrastructure Management service]
```
Comment (JoeAldinger) [inline scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc:45]: I don't know that I understand why this is being added both to the procedure and as an additional resource to the procedure/module. I'd consider omitting one.

**Review by JoeAldinger — COMMENTED** [review]
small suggestion and a question.

**Comment by mcornea** [comment]
lgtm

**Review by kelbrown20 — COMMENTED** [review]
Updates LGTM!

**Comment by kelbrown20** [comment]
@amolnar-rh Hello! All the other CPs are all set and merged, the CP for enterprise-4.12 failed so you will need to create a manual CP for that branch

**Pull request merged**